doc/contributing/model-layer.md

```diff
@@ -194,18 +194,18 @@ This means that we can call `asyncLoader.load()` many times without worrying abo
 
 `forceLoadX()` shouldn't be called directly - instead you should use `asyncLoader.load()` method - for example in `CatalogMemberMixin` we have
 
-- the abstract method `forceLoadMetadata`
-- `loadMetadata()` which wraps `asyncLoader.load()`
-- `loadMetadata()` can be called as many times as needed
-- See [CatalogMemberMixin example](#CatalogMemberMixin-example)
+-   the abstract method `forceLoadMetadata`
+-   `loadMetadata()` which wraps `asyncLoader.load()`
+-   `loadMetadata()` can be called as many times as needed
+-   See [CatalogMemberMixin example](#CatalogMemberMixin-example)
 
 A **correct** example:
 
 ```ts
 async function forceLoadX() {
-  const url = this.someObservableUrl;
-  const someData = await loadText(url);
-  runInAction(() => (this.someOtherObservable = someData));
+    const url = this.someObservableUrl;
+    const someData = await loadText(url);
+    runInAction(() => (this.someOtherObservable = someData));
 }
 ```
```

Comment:
indent_size=4 is even worse for code blocks in md files, but there's no prettier or editorconfig configuration that would fix this, and using 2 spaces would require patching mkdocs as the author of the parser used believes that lists in lists must be indented by 4 spaces exactly to be valid markdown.